tasks/buildin/webui/app/theme.css

```
@@ -26,237 +30,276 @@
+  --dicode-space-2xl:   3rem;     /* 48px */
+  --dicode-space-3xl:   4rem;     /* 64px */
+  --dicode-space-4xl:   6rem;     /* 96px */
+  --dicode-space-5xl:   8rem;     /* 128px */
 }
 
 /* ─── TYPOGRAPHY ──────────────────────────────────────────────────────────── */
 
 :root {
-  --font-sans:   system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
-  --font-mono:   "Fira Code", "Cascadia Code", ui-monospace, SFMono-Regular,
+  --dicode-font-sans:   system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)